assignments/array-methods.js

```diff
@@ -58,11 +58,20 @@ const runners = [
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs. Combine both the first and last names and populate a new array called `fullNames`. This array will contain just strings.
 let fullNames = [];
+runners.forEach(
+  function(run){
+    fullNames.push(`${run.first_name} ${run.last_name}`)
+  }
+);
+
 console.log(fullNames);
 
+
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runners' first names in uppercase because the director BECAME DRUNK WITH POWER. Populate an array called `firstNamesAllCaps`. This array will contain just strings.
 let firstNamesAllCaps = [];
+
```

Comment:
This could have been achieved in a similar fashion like you did with `forEach` method, but using `map` instead. Please read more about [array methods](https://javascript.info/array-methods) and [toUpperCase method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toUpperCase)